tests/rt.conf

```diff
@@ -209,33 +209,34 @@ RUN     | fv3_ccpp_esg_HAFS_v0_hwrf_thompson_debug
 RUN     | cpld_control                                                                                                        | standard | hera.intel orion.intel wcoss_dell_p3 | fv3 |
-#RUN     | cpld_control                                                                                                        | standard | cheyenne.intel                       | fv3 |
+# following two tests are not run on any platform
 #RUN     | cpld_control_12h                                                                                                    | standard | hera.intel orion.intel wcoss_dell_p3 | fv3 |
 #RUN     | cpld_restart                                                                                                        | standard | hera.intel orion.intel wcoss_dell_p3 |     | cpld_control_12h
 RUN     | cpld_2threads                                                                                                       | standard | hera.intel orion.intel wcoss_dell_p3 |     |
-#RUN     | cpld_2threads                                                                                                       | standard | cheyenne.intel                       |     |
 RUN     | cpld_decomp                                                                                                         | standard | hera.intel orion.intel wcoss_dell_p3 |     |
-#RUN     | cpld_decomp                                                                                                         | standard | cheyenne.intel                       |     |
 RUN     | cpld_satmedmf                                                                                                       | standard | hera.intel orion.intel wcoss_dell_p3 | fv3 |
-#RUN     | cpld_satmedmf                                                                                                       | standard | cheyenne.intel                       | fv3 |
 RUN     | cpld_ca                                                                                                             | standard | hera.intel orion.intel wcoss_dell_p3 | fv3 |
+#RUN     | cpld_control                                                                                                        | standard | cheyenne.intel                       | fv3 |
+#RUN     | cpld_2threads                                                                                                       | standard | cheyenne.intel                       |     |
+#RUN     | cpld_decomp                                                                                                         | standard | cheyenne.intel                       |     |
+#RUN     | cpld_satmedmf                                                                                                       | standard | cheyenne.intel                       | fv3 |
```

Comment:
These tests will be turned on in a future PR?

Reply:
Yes, I believe the plan is to support Cheyenne as Tier 1 for the ufs-cpld also. But I think we need to resolve whatever issue is causing the compile jobs to fail. Setting max jobs to 3 worked one time, but not another.